src/animation/tracks/StringKeyframeTrack.js

```diff
@@ -4,7 +4,16 @@ import { KeyframeTrack } from '../KeyframeTrack.js';
 /**
  * A Track that interpolates Strings
  */
-class StringKeyframeTrack extends KeyframeTrack {}
+class StringKeyframeTrack extends KeyframeTrack {
+
+	// No interpolation parameter because only InterpolateDiscrete makes sense.
```

Comment:
Instead of using the term "makes sense", can you please switch to "is valid"?

Reply:
updated this, also updated tests.